<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Аппроксимация</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableSolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="ChartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>671</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>189</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="DegreeRadioBtn">
       <property name="text">
        <string>Степенная</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="polynomRadioBtn">
       <property name="text">
        <string>Полином</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>187</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Степень полинома </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="polynomDegreeBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>481</y>
      <width>192</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sigma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sigmaEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="openFileBtn">
         <property name="text">
          <string>Открыть файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Расчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action">
   <property name="text">
    <string>фыв</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
